docs/developer-docs/latest/developer-resources/database-migrations.md

```diff
@@ -12,11 +12,15 @@ Database migrations exist to run one-time queries against the database, typicall
 Database migrations are experimental. This feature is still a work in progress and will continue to be updated and improved. In the meantime, feel free to ask for help on the [forum](https://forum.strapi.io/) or on the community [Discord](https://discord.strapi.io).
 :::
 
-## Understanding database migration files
+## Understand database migration files
```

Comment:
I kind of like "Understanding" better personally 😆

Reply:
I am trying to move away from -ing titles because they don't work well with translations, but I understand your preference. https://developers.google.com/style/headings